lib/util/git/index.ts

```diff
@@ -215,6 +217,23 @@ export async function setBranchPrefix(branchPrefix: string): Promise<void> {
   }
 }
 
+function setIgnoredAuthors(ignoredAuthors?: string[]): void {
+  config.ignoredAuthors = ignoredAuthors ?? [];
+}
```

Comment:
I think this can be inlined. I don't think we really need to split such small loc's to multiple functions.